pytest.ini

```diff
@@ -3,5 +3,6 @@ addopts = --cov . --pep8 --pylint --cov-report term --cov-report html --ds=sga_l
 norecursedirs = node_modules .git .tox static templates .* CVS _darcs {arch} *.egg
 pep8ignore =
    */migrations/* ALL
+   W503
```

Comment:
[PEP8 recommends line breaks before binary operators](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator), but pytest-pp8 W503 is opposite.